computed-types/src/types.ts

```diff
@@ -0,0 +1,14 @@
+import type {
+  FieldValues,
+  ResolverResult,
+  UnpackNestedValue,
+  ResolverOptions,
+} from 'react-hook-form';
+
+export type Resolver = (
+  schema: any,
```

Comment:
Can't find a good way to type the schema.

@kotarella1110 if you want to play with that one, _(no worries if you haven't time :))_